src/SFA.DAS.AssessorService.Data/RegisterQueryRepository.cs

```diff
@@ -374,6 +373,40 @@ public async Task<EpaContact> GetContactByContactId(Guid contactId)
             }
         }
 
+        public async Task<EpaContact> GetContactByEmail(string email)
+        {
+            var connectionString = _configuration.SqlConnectionString;
+
+            using (var connection = new SqlConnection(connectionString))
+            {
+                if (connection.State != ConnectionState.Open)
+                    await connection.OpenAsync();
+
+                var sql =
+                    "select Id, EndPointAssessorOrganisationId, Username,GivenNames, DisplayName, FamilyName, SigninId, SigninType, Email, Status, PhoneNumber " +
+                    " from Contacts where Email = @email";
+                var contact = await connection.QuerySingleOrDefaultAsync<EpaContact>(sql, new { email });
+                return contact;
+            }
+        }
+
+        public async Task<EpaContact> GetContactBySignInId(string signinId)
```

Comment:
Why is this returning an EpaContact and not a Contact?  And surely we have a GetContactBySignInId somewhere else?  Same for Email as well I guess?

Reply:
Probably as Mark said, that `EPA distinguished itself as being dappery, when there was a lot of EF still in use`
Assuming it's going to be a bit of a job to clean this up into 1 common object again